app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -65,9 +66,11 @@ class CtaViewModel @Inject constructor(
     private val userStageStore: UserStageStore,
     private val tabRepository: TabRepository,
     private val dispatchers: DispatcherProvider,
-    private val duckDuckGoUrlDetector: DuckDuckGoUrlDetector
+    private val duckDuckGoUrlDetector: DuckDuckGoUrlDetector,
+    private val appTheme: AppTheme,
 ) {
     val surveyLiveData: LiveData<Survey> = surveyDao.getLiveScheduled()
+    var canShowAutoconsentCta = false
```

Comment:
nit: This is modified inside `suspend` functions so being extra careful I would probably make this an `AtomicBoolean` just in case

Reply:
Done!